snaplets/heist/templates/users_add.tpl

```diff
@@ -0,0 +1,6 @@
+<apply template="base">
+  <h1>Add User</h1>
+  <br/>
+  <bind tag="action">/users/add</bind>
```

Comment:
@CristhianMotoche  Not in this case, but i was thinking maybe we could use the same form for other actions in the future, for example for updating the user it would help if the action is binded from a template that only applies the user form.

Reply:
@kinorius123 Ok, sounds like a great idea.